content/id/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Poseidon-Firmament - Sebuah Penjadwal Alternatif
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+**Rilis saat ini dari Penjadwal Poseidon-Firmament adalah rilis <code> alpha </code>.**
+
+Penjadwal Poseidon-Firmament adalah penjadwal alternatif yang dapat digunakan bersama penjadwal Kubernetes bawaan.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+
+## Pengenalan	
+
+Poseidon adalah sebuah layanan yang berperan sebagai pemersatu antara [Penjadwal Firmament](https://github.com/Huawei-PaaS/firmament) dengan Kubernetes. Penjadwal Poseidon-Firmament menambah kapabilitas penjadwal Kubernetes saat ini. Penjadwal ini menggabungkan kemampuan penjadwalan berbasis grafik jaringan grafis (_flow network graph_) baru bersama penjadwal Kubernetes bawaan. Penjadwal Firmament memodelkan beban kerja dan kelompok sebagai jaringan aliran dan menjalankan optimisasi aliran biaya-minimum kepada jaringan ini untuk membuat keputusan penjadwalan.
+
+Penjadwal ini memodelkan masalah penjadwalan sebagai optimasi berbasis kendala atas grafik jaringan aliran. Hal ini dicapai dengan mengurangi penjadwalan ke masalah optimisasi biaya-minimum aliran-maks. Penjadwal Poseidon-Firmament secara dinamis memperbaiki penempatan beban kerja.
+
+Penjadwal Poseidon-Firmament berjalan bersamaan dengan penjadwal Kubernetes bawaan sebagai penjadwal alternatif, sehingga beberapa penjadwal dapat berjalan secara bersamaan.
+
+## Keuntungan Utama
+
+### Penjadwalan grafik jaringan (_network graph_) berbasis penjadwalan Poseidon-Firmament memberikan beberapa keuntungan utama sebagai berikut:
+
+- Beban kerja (Pod) dijadwalkan secara kolektif untuk memungkinkan penjadwalan dalam skala besar.
+- Berdasarkan hasil tes kinerja yang luas, skala Poseidon-Firmament jauh lebih baik daripada penjadwal bawaan Kubernetes dilihat dari jumlah node meningkat dalam sebuah kluster. Hal ini disebabkan oleh fakta bahwa Poseidon-Firmament mampu mengamortisasi lebih banyak pekerjaan di seluruh beban kerja.
```

Comment:
```suggestion
- Berdasarkan hasil tes kinerja yang ekstensif, skala Poseidon-Firmament jauh lebih baik daripada penjadwal bawaan Kubernetes dilihat dari jumlah node meningkat dalam sebuah kluster. Hal ini disebabkan oleh fakta bahwa Poseidon-Firmament mampu mengamortisasi lebih banyak pekerjaan di seluruh beban kerja.
```

- Ekstensif is a valid Indonesian word -> https://kbbi.kemdikbud.go.id/entri/ekstensif

Reply:
done